sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java

```diff
@@ -284,7 +284,6 @@ public void setDecimal(int ordinal, Decimal value, int precision) {
         // keep the offset for future update
         Platform.putLong(baseObject, getFieldOffset(ordinal), cursor << 32);
       } else {
-
```

Comment:
@dongjinleekr, this comment was missed https://github.com/apache/spark/pull/25251/files#r307396454

Reply:
I'm sorry, my mistake;